src/ipyautoui/autoform.py

```diff
@@ -63,20 +46,6 @@ def bn_shownull(self):
     def bn_shownull(self, value):
         self._bn_shownull = value
 
-    def show_hide_bn_nullable(self):  # work with AutoObject only
```

Comment:
`show_hide_bn_nullable` checks to see if there are any nullable fields within the form - if there aren't then it doesn't display `bn_shownull` (as it wouldn't do anything....) 

not sure where this functionality is being replaced?

Reply:
Resolved here where we check for any null fields:

https://github.com/maxfordham/ipyautoui/blob/753095a18cdb991ba322b79862e2d35f1f5e0048/src/ipyautoui/autoform.py#L49-L67

Reply:
check_for_nullables could  be `autoobject` , with a dummy caller in `ShowNull` that gets overridden when it is inherited... but this is also good